pkg/tfbridge/info/autonaming.go

```diff
@@ -170,7 +171,45 @@ func ComputeAutoNameDefault(
 	if options.Transform != nil {
 		vs = options.Transform(vs)
 	}
-	if options.Randlen > 0 {
+	if defaultOptions.Autonaming != nil {
```

Comment:
Will auto-naming always be set on new engines?

Shouldn't `ModePropose` be the same as our prior default, or is it different.

Reply:
No, it will only be set if users provide an explicit autonaming configuration. It will stay nil for everyone else (majority of users).